src/core/eme/eme_manager.ts

```diff
@@ -170,7 +177,33 @@ export default function EMEManager(
                               value: initializationData });
       }
 
-      if (!handledInitData.storeIfNone(initializationData, true)) {
+      const lastKeyUpdate$ = new ReplaySubject<IKeyUpdateValue>(1);
+
+      // First, check that this initialization data is not already handled
+      const keyIds = initializationData.keyIds;
+      if (options.singleLicensePer === "content" &&
+          handledSessions.getLength() > 0 &&
+          keyIds !== undefined)
+      {
+        const firstSession = handledSessions.getAll()[0];
```

Comment:
Looks ugly, and now that I think of it, we could have a problem when still creating another session because we do not know one of the key id.

I will try to fix that

Reply:
Forgot what I just wrote.
As all sessions should have all key-ids, there should be no issue here.